mpv.py

```diff
@@ -479,6 +479,9 @@ def command(self, name, *args):
                 for arg in args if arg is not None ] + [None]
         _mpv_command(self.handle, (c_char_p*len(args))(*args))
 
+    def resume(self):
+        self._set_property("pause", False, proptype=bool)
```

Comment:
I don't think ```resume()``` is of much use if you cant ```pause()```. Is there some common case where you have a ```MPV``` instance that is paused by some way other than ```player.pause = True```?